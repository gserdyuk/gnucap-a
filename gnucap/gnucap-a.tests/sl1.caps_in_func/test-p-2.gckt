* capitals in parameter names
spice

.param  p1=1.1
.param  b=P1+1.1

v1 1 0 dc b

.print op v(1) 


.op

.end
