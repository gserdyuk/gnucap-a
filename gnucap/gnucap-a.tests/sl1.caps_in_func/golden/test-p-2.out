Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-04-09
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
#           v(1)      
 27.        2.2       
